marathon/resource_marathon_app.go

```diff
@@ -1097,11 +1097,11 @@ func mutateResourceToApplication(d *schema.ResourceData) *marathon.Application {
 	upgradeStrategy := &marathon.UpgradeStrategy{}
 
 	if v, ok := d.GetOk("upgrade_strategy.0.minimum_health_capacity"); ok {
-		upgradeStrategy.MinimumHealthCapacity = v.(float64)
+		upgradeStrategy.MinimumHealthCapacity = v.(*float64)
 	}
 
 	if v, ok := d.GetOk("upgrade_strategy.0.maximum_over_capacity"); ok {
-		upgradeStrategy.MaximumOverCapacity = v.(float64)
+		upgradeStrategy.MaximumOverCapacity = v.(*float64)
 	}
 
```

Comment:
Note: These were changed because `UpgradeStrategy` was made optional (`omitempty`) in go-marathon.

https://github.com/gambol99/go-marathon/pull/242